calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
+
+  let total = 0;
+  for (let i = 0; i < grades.length; i++) {
```

Comment:
If we initialize i as 0, and increment while i is < length, then at the end of the loop i will be incremented to an index value that is too large and doesn't exist within grades. We could change the loop condition to correct this:
```suggestion
  for (let i = 0; i < grades.length - 1; i++) {
```

Reply:
Thanks, but I think that since  we are using `<` the loop will not go one extra time.